backend/inspirehep/records/models.py

```diff
@@ -179,6 +179,7 @@ class RecordsAuthors(db.Model):
             "id_type",
             "record_id",
         ),
+        db.Index("ix_id_type_authors_id", "id_type", "author_id"),
```

Comment:
there is a global index for namespaces, so the table name should be included

Reply:
Sorry, forgot to add the file to git, thanks :)

Reply:
Not sure I understand, we can discuss it :)